contents/docs/libraries/hono.mdx

```diff
@@ -64,6 +67,10 @@ Cloudflare Pages uses its own middleware system that is different from Hono's mi
 
 </CalloutBox>
 
+<DetailSetUpReverseProxy />
```

Comment:
Hono is mostly a backend/server-side language (I think), so probably not necessary here